podtato-head-monolith/static/overview.html

```diff
@@ -0,0 +1,18 @@
+<html>
```

Comment:
Is this file even used?

Reply:
:man_shrugging: :smile: Same as others, let's analyze if and after we merge.